src/charm.py

```diff
@@ -93,6 +95,10 @@ def __init__(self, *args):
                 self.on.pushgateway_pebble_ready,
             ],
         )
+        # Enable log forwarding for Loki and other charms that implement loki_push_api
+        self._logging = LogProxyConsumer(
+            self, relation_name="log-proxy", log_files=[PUSHGATEWAY_LOG]
```

Comment:
don't we call this `logging` on all other charms?

Reply:
Yes, that's right. We use `log-proxy`